rest/model/src/main/java/io/syndesis/model/connection/Action.java

```diff
@@ -38,6 +38,8 @@
 @JsonIgnoreProperties(value = {"properties", "inputDataShape", "outputDataShape"}, allowGetters = true)
 public interface Action extends WithId<Action>, WithName, WithTags, WithConfigurationProperties, Serializable {
 
+    String VERSION = "1";
```

Comment:
Do you really want one version per object ? Sounds overkill to me, especially it is not guaranteed that object from different versions play well together. 

Couldn't we defined a 'model version' ?

Reply:
Another question: Who is responsible for updating the version number ? How it is guaranteed that its not forgotten to increment when the model changes ?